pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -292,7 +307,10 @@ def qtProjectGenerator(package, argv, extra):
     name = None
     kits = []
     try:
-        profiles = xml.etree.ElementTree.parse(os.path.join(expanduser('~'), ".config/QtProject/qtcreator/profiles.xml")).getroot()
+        if isWindows():
+            profiles = xml.etree.ElementTree.parse(os.path.join(os.getenv('APPDATA'), "QtProject/qtcreator/profiles.xml")).getroot()
```

Comment:
This is important, that windows can resolve the build presets and kits (kit ids) under windows.